src/mutation_data.cpp

```diff
@@ -20,27 +21,26 @@ typedef std::set<trait_id> TraitSet;
 TraitSet trait_blacklist;
 TraitGroupMap trait_groups;
 
+namespace
+{
+generic_factory<mutation_branch> trait_factory( "trait" );
+}
+
 std::vector<dream> dreams;
 std::map<std::string, std::vector<trait_id> > mutations_category;
 std::map<std::string, mutation_category_trait> mutation_category_traits;
-std::unordered_map<trait_id, mutation_branch> mutation_data;
+//std::unordered_map<trait_id, mutation_branch> mutation_data;
```

Comment:
Please don not check in commented out code. Either remove it or keep it as actual code.